### ISSUE TITLE
Document significant figures format in guidelines

### ISSUE THREAD

**Issue by mjskay** [transparentstats/guidelines#34]
I think this should be separate from the effect size FAQ, so maybe needs to be its own (short) FAQ / guideline describing our rationale and the function we use in all the other guidelines.

Do this after we decide on #33

**Comment by dragice** [comment]
Agreed. My guess is that the right format will depend on the case. Figures have semantics associated to them. For p for example there's this convention that we don't include the zero from the integer part. Also for intervals, [13.456, 13.448] seems reasonable but [13.456, 97.193] probably less so. There's also this trade-off between avoiding spurious precision vs. allowing people who reanalyze our data to verify their results. It's complicated. We should probably avoid APA-style rules such as "always report two sig digits", and instead illustrate different case scenarios.

A FAQ/guideline about graphic formats could be nice too.

**Comment by chatchavan** [comment]
For reference: previous discussion on sig. figure and R functions: https://github.com/transparentstats/guidelines/pull/32/files/2681012b345b797b766fa2e256a80008bf129dd3
